### DISCUSSION TITLE
RTX Circuit Breaker 2x16A Energy Meter (ZCB16-2P) Zigbee in ZHA

### DISCUSSION THREAD

**Discussion by tomkusz** [zigpy/zha-device-handlers#1575] [General]
Hello.
I have the problem with proper attachement Zigbee smart energy meter - RTX Circuit Breaker (_TZE200_fsb6zw01) with ZHA Zigpy (not Zigbee2mqtt) in Homeassistant. This stuff is the same as another energy meter DDS238-2 (_TZE200_ewxhg6o9). How to attach zhaquirk tuya ts0601_din_power.py to device _TZE200_fsb6zw01?

**Comment by tomkusz** [comment]
This energy meter  (_TZE200_fsb6zw01) attach to ZHA Zigpy with only 2 entities - lqi and rssi ... without zhaquirk ts0601_din_power.py connected.

**Reply by darckly** [reply]
Hello,

Did you try to create the custom quirk as described here https://github.com/zigpy/zha-device-handlers/issues/1275#issuecomment-1010788729, but add your ID to the MODELS_INFO list?

**Comment by Hedda** [comment]
FYI, this is now a "Device support request" under issues -> https://github.com/zigpy/zha-device-handlers/issues/1739
